test/load/allocation/grpc/configure-agones.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -e
+
+
+function main() {
+  echo "Make sure you have kubectl pointed at the right cluster"
+  tmp_dir=$(mktemp -d)
+
+  patch_controller="${tmp_dir}/patch-controller.yaml"
+  cat << EOF > "${patch_controller}"
+spec:
+  template:
+    spec:
+      affinity:
+        nodeAffinity:
+          \$retainKeys:
+            - requiredDuringSchedulingIgnoredDuringExecution
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: agones.dev/agones-system
+                operator: Exists
+      containers:
+      - name: agones-controller
+        resources:
+          limits:
+            cpu: "4"
+            memory: 4000Mi
+          requests:
+            cpu: "4"
+            memory: 4000Mi
+EOF
+
+  kubectl patch deployment -n agones-system agones-controller --patch "$(cat "${patch_controller}")"
```

Comment:
Should this be done through a `helm` install and use config options? (assuming it can be done that way?)

Reply:
I don't know if we can change the affinity via the helm install. You can definitely set the requests/limits using variables.

Reply:
Fair enough - if it can't be done that way, it may be something we have/want to pull into Helm at a later date.

Reply:
I figured out how to do it using helm so I've included those instructions as well.